cv19ResSupportV3/V3/Infrastructure/HelpRequestCallsEntity.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace cv19ResSupportV3.V3.Infrastructure
+{
+
+    [Table("help_request_calls")]
+        public class HelpRequestCallsEntity
```

Comment:
Should this be `HelpRequestCallEntity` instead of `HelpRequestCallsEntity`?

Reply:
Yes, probably...